templates/mod/expires.conf.erb

```diff
@@ -0,0 +1,11 @@
+ExpiresActive <%= scope.function_bool2httpd([@expires_active]) %>
+<%- if ! @expires_default.nil? and ! @expires_default.empty? -%>
+ExpiresDefault "<%= @expires_default %>"
+<%- end -%>
+<%- if ! @expires_by_type.nil? and ! @expires_by_type.empty? -%>
+<%-   [@expires_by_type].flatten.each do |line| -%>
```

Comment:
if we force this to [array], and flatten it, we don't need to check if it's nil/empty.